sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -39,6 +39,23 @@ If no `options` parameter is supplied, the report is exported using the default
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
 
+### getData(options?: Object): ReportObjectResultData[]
+
+Returns all currently displayed data from the report object.
+
+#### Arguments
+
+`options` is an optional options bundle for customizing the returned data. The following options are supported:
+
+- `formatData` specifies the format of the returned data.
+  - `true` returns all formatted data
+  - `false` returns all unformatted data
+  - `"datesOnly"` `default` returns only SAS date values as formatted and all other data is unformatted
+
```

Comment:
To make the list of items consistent, add a period after the word "data" for lines 51 and 52. For line 53, a couple of tweaks can help with localization.  "returns SAS date values as formatted data and returns all other values as unformatted data."

Reply:
Note that most of this text is identical to what is already documented for [`getSelectedData`](https://developer.sas.com/sdk/va/docs/api/ObjectHandle/#getselecteddataoptions-object-reportobjectresultdata).  @joylashley I assume all of the suggestions here would apply to that api doc as well?

Reply:
@brmorr, I didn't realize this was nearly identical to some existing text. Yes, let's apply the punctuation and the other updates in both places.